README.md

```diff
@@ -272,6 +272,68 @@ module.exports = {
 
 For long term caching use `filename: "[contenthash].css"`. Optionally add `[name]`.
 
+### Media Query
+
+When writing CSS with the help of a framework (such as [Bootstrap](https://getbootstrap.com/) or [Foundation](https://foundation.zurb.com/sites.html)) and with a modular design pattern your emited CSS files will probably contain many media queries. This is quite bad for mobile users who have to load all desktop specific CSS which they will never need.
+
+To improve this you can use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin) which plays together well with the mini-css-extract-plugin. It will extract the defined media queries and emit them as separate files (or inject into existing chunks if working with dynamic imports).
+
+Afterwards a mobile user only needs to load this
+
```

Comment:
Please add 'useful/expressive' filenames for context in the following format ```**`useful.css`** ``` to all code examples, e.g ``` `**webpack.config.js**` ```to the config examples

Reply:
done!